lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -675,6 +676,17 @@ class DocumentImpl extends NodeImpl {
     });
   }
 
+  createTreeWalker(root, whatToShow, filter) {
+    if (!("_ownerDocument" in root)) throw new TypeError("First argument to createTreeWalker must be a Node");
```

Comment:
You're both wrong :). I updated that file to use the isImpl check for Node. If you rebase you should have a better idea of what to copy and paste from.